src/commands/generate/index.js

```diff
@@ -0,0 +1,162 @@
+import _ from "lodash";
```

Comment:
Import only parts that you need for tree-shaking

Reply:
Unless runtime does tree-shaking of sorts, I don't think it's happening with our setup. Also, wouldn't it just happen one import down inside lodash?

Reply:
It does not because `lodash` as is is not three-shakable.
Import concrete functions directly `import isEmpty from lodash/isEmpty`

Reply:
Again, tree-shaking is likely not happening. Anyway, in case runtime might benefit from it regardless, let's switch.